src/server/config.js

```diff
@@ -6,71 +6,30 @@ var fs = require('fs'),
 var config = {};
 var simulationFilePath;
 
+// module properties
+[
+    'liveReload',
+    'platform',
+    'platformRoot',
+    'forcePrepare',
+    'projectRoot',
+    'server',
+    'telemetry',
+    'simHostOptions',
+    'xhrProxy',
+    'touchEvents'
+].forEach(function (prop) {
```

Comment:
Uh! you're right, I missed that! what do you think about keeping the optional value, and change it in this way:
```
[
    { name: 'liveReload'},
    { name: 'platform' },
    { name: 'platformRoot' },
    { name: 'forcePrepare' },
    { name: 'projectRoot', single: true },
    { name: 'server', single: true },
    { name: 'telemetry', single: true },
    { name: 'xhrProxy', single: true },
    { name: 'simHostOptions' },
    { name: 'touchEvents' }
].forEach(function (prop) {
    Object.defineProperty(module.exports, prop.name, {
        get: function () {
            return getValue(prop.name);
        },
        set: function (value) {
            setValue(prop.name, value, prop.single);
        }
    });
});
```